conformance/conformance.proto

```diff
@@ -57,6 +57,14 @@ enum WireFormat {
   JSON = 2;
 }
 
+enum TestCategory {
+  BINARY_TEST = 0;  // Test binary wire format.
+  JSON_TEST = 1;  // Test json wire format.
+  JSON_IGNORE_UNKNOWN_PARSING_TEST = 2;  // Similar to JSON_TEST. However,
```

Comment:
Add a link to https://developers.google.com/protocol-buffers/docs/proto3#json_options and point out this is an optional feature of proto3 JSON.